File: implementation/1316/1316.py
import sys
input = sys.stdin.readline

def checker(s):
	_set = set()
	for i in range(len(s)):
		if i > 0 and s[i - 1] == s[i] or not s[i] in _set:
			_set.add(s[i])
			continue
		return False
	return True

count = 0
for _ in range(int(input())):
	if checker(input()):
		count += 1
print(count)

File: until_20221109/1316.c
#include <stdio.h>

int main() {
    int n, count = 0;
    scanf("%d", &n);
    for (int i = 0; i < n; i++) {
        count += checker();
    }
    printf("%d\n", count);
}

int checker() {
    int n;
    char array[100];
    scanf("%s", &array);
    for (int i = 0; i < sizeof(array); i++) {
        
    }
    return ;
}